### DISCUSSION TITLE
Does pssh or dsh work with brew

### DISCUSSION THREAD

**Discussion by DavidNwanze** [Homebrew/discussions#2441] [Everyday usage]
Hi there im trying to set up an automated system for a group of mac's for a project of time and everything went well so far got the ssh key added host files made but when using pssh or dsh I always get the error Exited with error code 127 and zsh:1: command not found brew. I have looked everywhere for answers I am starting to think pssh is unable to run brew commands. Doest anyone know how to fix this

**Comment by SMillerDev** [comment]
Is the brew prefix added to the path there?
